### ISSUE TITLE
last.fm example accuracy issues

### ISSUE THREAD

**Issue by zluo01** [benfred/implicit#301]
I run the last.fm example provided in the README. For the ALS, I get loss = nan ? Is this expected for the example ?

Thank you

**Comment by benfred** [comment]
You shouldn't get a NaN value - are you running the example file directly? How are you calculating loss - the example file doesn't do that iirc.
